sklearn/feature_selection/from_model.py

```diff
@@ -183,10 +187,23 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
     `threshold_`: float
         The threshold value used for feature selection.
     """
-    def __init__(self, estimator, threshold=None, prefit=False):
+    def __init__(self, estimator, threshold=None, prefit=False,
+                 max_n_features=None):
         self.estimator = estimator
         self.threshold = threshold
         self.prefit = prefit
+        self.max_n_features = max_n_features
+
+    def _check_params(self, X, y):
+        X, y = check_X_y(X, y)
```

Comment:
Do we need this? As long as the wrapped estimator considers these valid, do we need to care in the metaestimator?